api/core/television/television.sendCommand.js

```diff
@@ -0,0 +1,30 @@
+const Promise = require('bluebird');
+const queries = require('./television.queries.js');
+
+module.exports = function sendCommand(functionName, params) {
+    return getDeviceType(params)
+      .then((deviceType) => {
+
+          if(!gladys.modules[deviceType.service] || !gladys.modules[deviceType.service].television)
+                return Promise.reject(new Error(`television : Module ${deviceType.service} does not exist or does not handle television.`));
+         
+          if(typeof gladys.modules[deviceType.service].television[functionName] != 'function') {
+            console.log(`television : Module ${deviceType.service} does not have function ${functionName}`)
+            return Promise.reject();
+          } 
+
+          params.deviceType = deviceType;
+        
+          // calling television module
+          return gladys.modules[deviceType.service].television[functionName](params);
+        });
+};
+
+function getDeviceType(params){
+    if(params.device) return gladys.utils.sqlUnique(queries.getDeviceTypeByDeviceIdOrByDeviceTypeId, [params.device, params.device, null, null]);
+    else if(params.room) return gladys.utils.sqlUnique(queries.getTelevisionDeviceTypeByRoom, [params.room]);
+    else if(params.deviceType) return gladys.utils.sqlUnique(queries.getDeviceTypeByDeviceIdOrByDeviceTypeId, [null, null, params.deviceType, params.deviceType]);
```

Comment:
In the music API, params.devicetype is lowercase! Maybe we should do the same here ?

Reply:
This a problem already raised on the forum [here](https://community.gladysproject.com/t/devicetype-ou-devicetype/2881), and present in other Gladys modules.
What could be done to solve it definitively?